packages/celotool/src/lib/geth.ts

```diff
@@ -538,30 +541,36 @@ export enum TestMode {
   Data = 'data',
   Transfer = 'transfer',
   StableTransfer = 'stable_transfer',
+  ContractCall = 'contract-call',
```

Comment:
maybe use 'contract_call' to use the same format as above?

Reply:
Updated.